config/Dockerfiles/tests.d/openstack/06_os-network

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash -xe
 
-# Verify the openstack volume provisioning with template data
-# distros.exclude: none
+# Verify the openstack network provisioning with template data
+# distros.exclude: centos7 fedora29 fedora30
```

Comment:
Why are we disabling this test?  In general, if we're disabling tests can they just be deleted?

Reply:
@14rcole We are disabling the test since it is being tested as part of another test.
We are not deleting tests because if we would like to enable it in future for some other variant it would be useful

Reply:
Can we make some not of when we disabled the test and why?  If we disable a test because it's not necessary and that test is still sitting around 5 releases later, it may be nice to know we can clean it up.  We can always grab the file from old release branches if we need to restore it a long time down the line

Reply:
@14rcole I think what you said makes sense I will delete the disabled test.